sample-queries/sample-queries.json

```diff
@@ -2983,6 +2983,15 @@
     }
   ],
   "TeamsAppSampleQueries": [
-      
+            {
+                "id": "fe15123a-507a-45d8-aa29-e5b25f5c0ac7",
+                "category": "Microsoft Teams",
+                "method": "GET",
+                "humanName": "list channel messages",
+                "requestUrl": "/beta/teams/{team-id}/channels/{channel-id}/messages",
+                "docLink": "https://docs.microsoft.com/en-us/graph/api/channel-list-messages?view=graph-rest-beta&tabs=http",
+                "tip": "This query requires a team id and a channel id. To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams. To find the channel id of channels you belong to, stay in user (delegated permissions) mode and run: GET https://graph.microsoft.com/beta/teams/{team-id}/channels. Also ensure that the ChannelMessage.Read.Group permission is in your manifest.json.",
```

Comment:
This sentence isnt really applicable 'Also ensure that the ChannelMessage.Read.Group permission is in your manifest.json.'